packages/http-client-csharp/eng/pipeline/publish.yml

```diff
@@ -0,0 +1,45 @@
+trigger: none
+pr: none
+
+extends:
+  template: /eng/pipelines/templates/1es-redirect.yml
+  parameters:
+    variables:
+      BlobFeedUrl: https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json
```

Comment:
Is this used?

Reply:
no, I was just checking in a file to test the pipeline triggers